src/Actions.ts

```diff
@@ -72,18 +72,18 @@ export class Actions {
         // Create a new poll passing in the poll author and the other params
         const poll = Poll.slashCreate(`<@${req.body.user_id}>`, req.body.text.split("\n"));
         try {
-            await this.postMessage(req.body.channel_id, "A poll has been posted!", poll.getBlocks());
+            this.postMessage(req.body.channel_id, "A poll has been posted!", poll.getBlocks());
             res.send();
```

Comment:
This one should probably remain in case `this.postMessage` throws, or move `res.send()` in a `.then()` handler.

Reply:
Good point. I guess technically any of the post messages could throw, but this one is most important as I just tested if it does throw and there isn't an await slack gets the `res.send();` and thinks everything is fine therefore the user has no idea what's going on.